sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_RecognizeHealthcareEntities.md

```diff
@@ -29,7 +27,13 @@ To recognize healthcare entities in multiple documents, call `StartAnalyzeHealth
                         increased symptoms and family history and history left main disease with total occasional of his RCA was referred for revascularization with open heart surgery.";
 
     string document2 = "Prescribed 100mg ibuprofen, taken twice daily.";
+```
 
+## Analyze Documents
+
+To recognize healthcare entities in multiple documents, call `StartAnalyzeHealthcareEntities` on an `IEnumerable` of strings.  The result is a Long Running operation of type `AnalyzeHealthcareEntitiesOperation` which polls for the results from the API.
+
+```C# 
```

Comment:
unfortunately, we can't have the code like this :( 
It has to always use the `Snippet:xx` syntax. How it works is that the `samples.cs` classes (the ones you are modifying in this PR` have the code and with `snippets` we insert that code into the `md` file.
We do this to avoid having this code be old or not updated to the latest reference of the library.

As an example, you can see how the comments in this `md` file are not the same comments you added into the `cs` classes. There are some missing, and this wouldn't happen if you use snippets